airgun/views/product.py

```diff
@@ -0,0 +1,72 @@
+from widgetastic.widget import Select, Text, TextInput, View
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    ActionsDropdown,
+    ConfirmationDialog,
+    EditableEntry,
+    EditableEntrySelect,
+    ReadOnlyEntry,
+)
+
+
+class ProductsTableView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h2[contains(., 'Products')]")
+    new = Text("//button[contains(@href, '/products/new')]")
+    edit = Text(
+        "//td/a[contains(@ui-sref, 'product.repositories') and "
+        "contains(@href, 'products')]"
+    )
+    repo_discovery = Text("//button[contains(.,'Repo Discovery')]")
+    actions = ActionsDropdown("//td//div[contains(@class, 'btn-group')]")
```

Comment:
Could you please point me where's that actions list inside table cell? I'm a bit confused, but that's how table view looks for me (and this locator doesn't work there for obvious reasons):
<img width="696" alt="image" src="https://user-images.githubusercontent.com/11719030/39777471-4a88e26a-530c-11e8-82df-2f66e4dadb52.png">

Reply:
will rename class work for you?

Reply:
because I was talking about 'select action' dropdown obviously

Reply:
You misunderstood me - key words were "inside table cell". Your locator points to some `//td`, but there's no table surrounding that actions dropdown. Wrong locator then?